monkey/monkey_island/cc/setup/certificate/certificate_setup.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import os
+
+from common.utils.exceptions import InsecurePermissionsError
+from monkey_island.cc.services.utils.file_permissions import has_sufficient_permissions
+from monkey_island.cc.setup.island_config_options import IslandConfigOptions
+
+logger = logging.getLogger(__name__)
+
+
+def setup_certificate(config_options: IslandConfigOptions) -> (str, str):
+    crt_path = config_options.crt_path
+    key_path = config_options.key_path
+
+    # check paths
```

Comment:
If you ever find yourself writing comments like this, you should stop and wrap the relevant code in it's own function. The function name then does the same job as your comment.